pkg/operator/sync.go

```diff
@@ -830,9 +831,25 @@ func (optr *Operator) getGlobalConfig() (*configv1.Infrastructure, *configv1.Net
 	if err != nil && !apierrors.IsNotFound(err) {
 		return nil, nil, nil, nil, err
 	}
+	if proxy != nil && proxy.Status != (configv1.ProxyStatus{}) {
+		proxy.Status.NoProxy = removeEtcdNoProxyConfig(proxy.Status.NoProxy)
+	}
 	return infra, network, proxy, dns, nil
 }
 
+// removeEtcdNoProxyConfig temporary removal of etcd NoProxy records
```

Comment:
nit: s/temporary remove of/temporarily remove/

Reply:
done